src/wp-content/themes/twentytwelve/header.php

```diff
@@ -36,8 +36,18 @@
 <div id="page" class="hfeed site">
 	<header id="masthead" class="site-header">
 		<hgroup>
-			<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
-			<h2 class="site-description"><?php bloginfo( 'description' ); ?></h2>
+			<?php 
```

Comment:
```suggestion
			<?php
```

PHPCS fix

Reply:
@mukeshpanchal27 Thanks for reviewing the PR, I have implemented the suggestions.